packages/react-server/src/render/render.tsx

```diff
@@ -35,9 +35,9 @@ export function createRender(render: RenderFunction) {
     const asyncAssetManager = new AsyncAssetManager();
     const acceptsLanguages = ctx.acceptsLanguages && ctx.acceptsLanguages();
     const locale = Array.isArray(acceptsLanguages) ? acceptsLanguages[0] : 'en';
-    const app = render({...ctx, locale});
 
     try {
+      const app = render({...ctx, locale});
```

Comment:
Couple things here:

- The `render` call needs to be in the try/catch block error handling
- `return response;` doesn't do anything in koa middlewares. It's been removed cc/ @cartogram in case it does have some weird side-effecty use case.

Reply:
@ismail-syed its okay to remove `return response`.